app-modules/ai/resources/views/components/assistant.blade.php

```diff
@@ -459,6 +459,22 @@ class="ml-auto flex h-6 w-6 cursor-pointer items-center gap-2 rounded-md p-1 tex
                     <form x-on:submit.prevent="sendMessage">
                         <div
                             class="w-full overflow-hidden rounded-xl border border-gray-950/5 bg-gray-50 shadow-sm dark:border-white/10 dark:bg-gray-700">
+                            @if ($this->files)
+                                <div class="flex flex-row justify-start px-4 py-2">
+                                    @foreach ($this->files as $key => $file)
+                                        <div class="mx-2 flex items-center">
+                                            <x-filament::badge>
+                                                {{ $file['name'] }}
+                                                <x-slot
+                                                    name="deleteButton"
+                                                    label="Remove uploaded file {{ $file['name'] }}"
+                                                    wire:click="removeUploadedFile({{ $key }})"
+                                                ></x-slot>
+                                            </x-filament::badge>
+                                        </div>
+                                    @endforeach
+                                </div>
```

Comment:
I expect this produces the same results

```suggestion
                                <div class="flex items-center justify-start gap-3 px-4 py-2">
                                    @foreach ($this->files as $key => $file)
                                        <x-filament::badge>
                                            {{ $file['name'] }}
                                            <x-slot
                                                name="deleteButton"
                                                label="Remove uploaded file {{ $file['name'] }}"
                                                wire:click="removeUploadedFile({{ $key }})"
                                            ></x-slot>
                                        </x-
                                    @endforeach
                                </div>
```